package.json

```diff
@@ -72,12 +75,13 @@
     "mocha": "cd packages/terra-functional-testing && npm run mocha",
     "pack": "webpack -p",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
+    "prepare-for-release": "npm run terra -- prepare-for-release",
```

Comment:
In the other repos would we use this syntax?

```
    "prepare-for-release": "terra prepare-for-release",
```

Reply:
Yes.  This is just to get around mono repo weirdness.